lib/server.js

```diff
@@ -274,7 +390,7 @@ function _createBlockStatus(blockHeight, ledgerNode, voter, callback) {
         if(block.consensusMethod !== 'Continuity2017') {
           return callback(new BedrockError(
             'No Continuity2017 status for block found.',
-            'NotFound', {block, httpStatus: 404, public: true}));
+            'NotFoundError', {block, httpStatus: 404, public: true}));
```

Comment:
Heh, we're going to need to update a lot of legacy code that uses `NotFound` instead of `NotFoundError` if we want to head this way. Thoughts?

Reply:
I don't know about changing legacy code, but given that `NotFoundError` is being used widely in the ledger code already, I thought a decision had already been made to use `NotFoundError` in new modules.  And I suppose we would continue doing `NotFound` in older modules where it is already be used.

Reply:
@mattcollier, Ok, I didn't realize/remember that we had made that decision. That's fine.